cypress/integration/user.spec.js

```diff
@@ -26,3 +18,48 @@ describe('User', () => {
     cy.get('[data-testid=orcid-info]').contains('https://sandbox.orcid.org/0000-0001-2345-6789');
   });
 });
+
+describe('User connects to their Authority', () => {
+  beforeEach('Given that the user is logged in:', () => {
+    cy.visit('/');
+    cy.mocklogin();
+  });
+
+  it('The user should be able to connect to an authority on the start page when no authority is connected', () => {
+    cy.get('[data-testid=logo]').click({ force: true });
+
+    // wait for modal to open
+    cy.wait(3000);
```

Comment:
trodde greia med cypress var at den prøver å hente modal helt til den gir opp, slik at man ikke må legge inn egne waits?

Reply:
that is correct. will fix